brave/brave-core
# Added custom order sort to bookmarks side panel

fix https://github.com/brave/brave-browser/issues/34776

![Screenshot from 2023-12-13 15-24-10](https://github.com/brave/brave-core/assets/6786187/969e5d9a-83f1-43cd-9797-1113bbc09771)

It would be great to contribute this change to upstream.
Left comment - https://bugs.chromium.org/p/chromium/issues/detail?id=1444154#c11


Resolves 



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open bookmarks side panel and set `Custom order` sort filter
2. Open any folder in side panel and check items order is same with bookmakrs bar or bookmarks page
